app/actions/ControlActions.js

```diff
@@ -451,17 +454,34 @@ export const REVOKETICKETS_ATTEMPT = "REVOKETICKETS_ATTEMPT";
 export const REVOKETICKETS_FAILED = "REVOKETICKETS_FAILED";
 export const REVOKETICKETS_SUCCESS = "REVOKETICKETS_SUCCESS";
 
-export const revokeTicketsAttempt = (passphrase) => (dispatch, getState) => {
+export const revokeTicketsAttempt = (passphrase) => async (
+  dispatch,
+  getState
+) => {
   dispatch({ type: REVOKETICKETS_ATTEMPT });
-  return wallet
-    .revokeTickets(sel.walletService(getState()), passphrase)
-    .then((revokeTicketsResponse) => {
-      setTimeout(() => {
-        dispatch(getStakeInfoAttempt());
-      }, 4000);
-      dispatch({ revokeTicketsResponse, type: REVOKETICKETS_SUCCESS });
-    })
-    .catch((error) => dispatch({ error, type: REVOKETICKETS_FAILED }));
+  const walletService = sel.walletService(getState());
+  const accounts = sel.balances(getState());
+  const accountUnlocks = [];
+  accounts.map((acct) => {
+    // just skip if imported account.
+    if (acct.accountNumber === Math.pow(2, 31) - 1) {
```

Comment:
Use `>=` instead of `===` because we could eventually have more than a single imported account.

Also, would be better to set this as a constant (either locally here or on `constants/decred.js`) so this is less magical.